spec/controllers/account/contacts_controller_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe Account::ContactsController, type: :controller do
+  describe "GET #index" do
+    it "returns a 200 status code" do
+      @super_admin = FactoryBot.create(:user, :super_admin)
```

Comment:
try to avoid usage of instance variables in specs
prefer to use `#let` or `#let!` to declare data

Reply:
Made changes.